.github/workflows/commit_check.yaml

```
@@ -3,65 +1,167 @@
-        - run: isort . --diff --check-only
-        - run: black --check --diff .
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: 'pip'
+          
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements/dev.txt
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)